src/main/java/core/basesyntax/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
+    @Override
+    public List<String> read(String fileName) {
+        try {
```

Comment:
Consider using try-with-resources to ensure that all resources are closed properly, even if an IOException is thrown. This is particularly important when dealing with file operations.